### DISCUSSION TITLE
Car Information Tab

### DISCUSSION THREAD

**Discussion by Kay0002** [BimmerGestalt/AAIdrive#512] [Q&A] [answered]
Hi,

Is the intent of the information tab to show real-time data or store the data from the last time the car and app had a connection?

I ask because I just noticed that my information tab is showing info (i.e. location) from January 19th, which is the last time my phone was connected to the car. I don't see a way to refresh the information.

**Comment by hufman** [comment, marked as answer]
Indeed, that tab is meant to show after-drive summaries (the location, fuel economy, window status, and other advanced car information). It can show some live-updating information but the intention is for other apps to display live performance data in a more optimized fashion.
This cached data updates automatically while it is connected to the car. AAIdrive does not use the BMW Cloud API and so can only update the information while it is directly connected to the car.

**Reply by Kay0002** [reply]
Ok, I see. If there is an addon that reports live status for door (open/closed), windows (open/closed), sunroof (open/ closed), tailgate (open/closed), fuel level, remaining fuel distance, tire pressure, lock/unlock status, and location that would be amazing! If you could also control the lock and unlock status that would be awesome. 

The offical bmw app is very unreliable in my opinion.

Thanks for the reply!

**Reply by hufman** [reply]
Thanks for the suggestion! The car's Cloud API is the single source of truth for remote information about the car, so there could be no better information than what MyBMW shows you. Home Assistant, for example, has a plugin to view the same information, allowing for greater customizability in presentation, but the data is the same as what is shown in MyBMW.
